### PR TITLE
Refactoring pipeline start tests.

### PR DESCRIPTION
**Pull request by 16yuki0702**
# Changes
Integrating almost test cases into same table driven test cases.
(some test cases are still remain separately)

Removed ```pkg/cmd/pipeline/logs_testutil.go```

Related issue is https://github.com/tektoncd/cli/issues/520
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [ ] Run the code checkers with `make check`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
This changes don't affect any features.
```

### CODE REVIEW

File: pkg/cmd/pipeline/start_test.go [hunk 1]
```diff
@@ -1898,665 +685,1655 @@ func Test_start_pipeline_imageRes_withExistingRes_createNew(t *testing.T) {
...
-
-	cs, _ := test.SeedTestData(t, pipelinetest.Data{
-		Pipelines: []*v1alpha1.Pipeline{
-			tb.Pipeline(pipelineName, "ns",
-				tb.PipelineSpec(
-					tb.PipelineDeclaredResource("imageres", "image"),
-				),
-			),
-		},
+func Test_start_pipeline(t *testing.T) {
```
Comment (16yuki0702) [inline pkg/cmd/pipeline/start_test.go]: The test cases below from here are doing its own checks.
So it seems to be hard integrating into same table driven test cases and there is no merit doing it, I left these as these.

File: pkg/cmd/pipeline/start_test.go [hunk 2]
```diff
@@ -285,1118 +420,22 @@ func Test_start_pipeline_interactive(t *testing.T) {
...
-						tb.PipelineTaskInputResource("workspace", "git-repo"),
-						tb.PipelineTaskOutputResource("workspace", "source-repo"),
-					),
-					tb.PipelineTask("then-check", "check-stuff-file-exists",
-						tb.PipelineTaskInputResource("workspace", "git-repo", tb.From("first-create-file")),
-					),
-				),
-			),
-		},
+	// Without pipeline resource
```
Comment (danielhelfand) [inline pkg/cmd/pipeline/start_test.go]: Could you please clarify what this comment means? There is a pipeline resource that is part of `cs2`.

File: pkg/cmd/pipeline/start_test.go [hunk 3]
```diff
@@ -1433,77 +472,16 @@ func Test_start_pipeline_newGitRes_noExistingRes(t *testing.T) {
...
-		})
-	}
-}
-
-func Test_start_pipeline_newMultipleRes_noExistingRes(t *testing.T) {
-
-	pipelineName := "gitpipeline"
-
-	cs, _ := test.SeedTestData(t, pipelinetest.Data{
+	// Without pipeline resource of multiple
```
Comment (danielhelfand) [inline pkg/cmd/pipeline/start_test.go]: Can you clarify what this comment means?

File: pkg/cmd/pipeline/start_test.go [hunk 4]
```diff
@@ -1898,665 +685,1655 @@ func Test_start_pipeline_imageRes_withExistingRes_createNew(t *testing.T) {
...
+						return errors.New("pipelinerun not found")
+					}
+
+					c.Close()
+					return nil
+				},
+			},
+		},
+		{
+			name:               "Create new multiple pipeline resource",
```
Comment (danielhelfand) [inline pkg/cmd/pipeline/start_test.go]:
```suggestion
			name:               "Create multiple, new pipeline resources",
```

**Comment by 16yuki0702** [comment]
@danielhelfand Thank for your review!! I've fixed :pray:

**Pull request merged**